app/components/ai-reporting/preview/index.ts

```
@@ -0,0 +1,258 @@
+            this.errorScreenDesc = this.intl.t(
+              'reportModule.noSelectedColumnsDesc'
+            );
+          }
+
+          this.filterDetails = this.reportPreview?.filter_details || [];
+        }
+      } catch (err) {
+        const error = err as AdapterError;
+        let errMsg = this.intl.t('pleaseTryAgain');
+
+        if (error.errors && error.errors[0]?.detail) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)